test/TestData.js

```diff
@@ -430,9 +431,7 @@ const TestData = {
   SampleCorrect: {
     owner: "Max Novelli",
     description: "This is a very important sample",
-    sampleCharacteristics: {
```

Comment:
I think this should be kept

Reply:
Reversed it back with the correct value. 
Thanks for pointing out this